src/store/properties.ts

```diff
@@ -492,6 +492,27 @@ export class Properties extends VuexModule {
         }
       });
   }
+
+  get showAdvancedOptionsPanel() {
+    return (image: string) => {
+      const labels = this.workerImageList[image];
+      return labels ? labels.advancedOptionsPanel !== "False" : true;
+    };
+  }
+
+  get showAnnotationConfigurationPanel() {
+    return (image: string) => {
+      const labels = this.workerImageList[image];
+      return labels ? labels.annotationConfigurationPanel !== "False" : true;
+    };
+  }
```

Comment:
I you want to make these checks case insensitive, you can do:
`myLabel.toLowerCase() !== "false"` instead of `myLabel !== "False"`